packages/crypto/src/key_pair_ed25519.ts

```diff
@@ -50,7 +55,8 @@ export class KeyPairEd25519 extends KeyPairBase {
     }
 
     toString(): string {
-        return `ed25519:${this.secretKey}`;
+        const extendedSK = baseEncode(new Uint8Array([...baseDecode(this.secretKey), ...this.publicKey.data]));
```

Comment:
I think we can optimize this a bit by calculating the value of`extendedSK` in the constructor and returning the value in `toString()`